frontend/angular.json

```diff
@@ -37,6 +37,7 @@
               "src/assets"
             ],
             "styles": [
+              "src/assets/theme.css",
```

Comment:
Shouldn't we throw out one of the packages so that we don't have too many?

Reply:
Ja ich wollte ja Material raus hauen, aber ich werde das im 2. PR machen